oak-core/src/main/java/org/apache/jackrabbit/oak/query/QueryEngineSettings.java

```diff
@@ -59,7 +61,9 @@ public class QueryEngineSettings implements QueryEngineSettingsMBean, QueryLimit
 
     public static final String OAK_QUERY_PREFETCH_COUNT = "oak.prefetchCount";
 
-    public static final int DEFAULT_PREFETCH_COUNT = Integer.getInteger(OAK_QUERY_PREFETCH_COUNT, 0);
+    public static final String FT_NAME_PREFETCH_FOR_QUERIES = "FT_PREFETCH_OAK-9893";
```

Comment:
I wonder if it might make sense to have all the Feature Toggles listed in a separate class (just to have a readily available list of feature toggles available in oak).

Just a thought - not a strong suggestion.

Reply:
I expect that we add more and more feature toggles in various components. It would limit modularity if we list them in a central place I assume.